docs/installation-deployments.md

```diff
@@ -39,7 +39,20 @@ node build/start-server.js
 ```
 
 # Deploy using Docker
-Docs to be written, please help!
+Run using Docker directly:
+```
+sudo docker run -d -p 8787:8787 portkeyai/gateway
```

Comment:
- Can we please remove sudo from the command? 
- And we can also add latest tag to make it more explicit `portkeyai/gateway:latest`
- Can you also link the docker image url as a part of comment: https://hub.docker.com/r/portkeyai/gateway

Reply:
No way